### DISCUSSION TITLE
BOM

### DISCUSSION THREAD

**Discussion by jeremylaratro** [peterantypas/maiana#74] [General]
Hello,
First off - this is a really awesome project, so thank you for sharing it. I recently bought my first boat (small, and cheap, but a boat!) and I would like to add AIS, but I don't have the money (college student) and I love building electronics so when I can build something, that is the route I naturally go. 

My main question right now is, is there a BOM available? I see where the gerbers are and a bunch of other info, firmware, etc, but I only see a mention of the BOM in relation to the kits/mailing list. 
I already have a nice stock of SMT components so I was planning on ordering the gerbers and putting as much of the boards together as I have the parts for, then making an order for anything I don't have. 

Also - when I upload the gerbers to pcbway, there is a drill hole that is about as large as the board itself. Obviously, I just delete it/hide it, it doesn't seem to have any critical effect on the board, but I was wondering which company you used as I have had experiences in the past where a board that works great fro JL is manufactured differently from PCBway, vice versa.  I saw you mentioned JLPcb, but not sure if that is currently true. 

Thanks!

**Comment by peterantypas** [comment]
Look under Latest/CAD/Board-10.5 for a BOM.

Do not attempt to build any other.

**Reply by peterantypas** [reply]
Also, JLCPB is my fab house of choice. I stopped working with PCBWay years ago.

**Comment by peterantypas** [comment]
Oh, one more thing: You won't be able to source Si446x transceivers that easily. Better secure BOM first, before you do anything else.

**Reply by jeremylaratro** [reply]
I got busy and totally forgot about this comment until I looked through my github history. Thanks for the answers and tips! Next time I make an order for PCBs this will be on my list. Will definitely post an update here if so
